salt/orchestrate/aws/rds.sls

```diff
@@ -59,6 +61,22 @@ set_{{ name }}_master_password_in_vault:
 {% set master_pass = master_pass.data.value %}
 {% endif %}
 
+{# If engine is postgres and version is 10, then use major version only for
+   parameter group family. All others use major.minor. #}
+{% if engine == 'postgres' and engine_version.startswith('10') %}
```

Comment:
This is fragile as the current version of Postgres is now at 12, so it will fail for versions 11, 12, etc. Maybe verify on something like `len(engine_version.split('.')[0]) > 1`

Reply:
I've just pushed a commit that I think will fix this. It uses `int(engine_version.partition('.')[0]) > 9`.